.github/workflows/nightly.yml

```diff
@@ -357,6 +360,16 @@ jobs:
             exit 1
           fi
 
+      - name: Don't log raw data
+        shell: bash
+        timeout-minutes: 20
+        run: |
+          if rg '^' "${{ matrix.safe_path }}"/*/*/logs | awk 'length($0) > 15000 { print; exit 1 }'
```

Comment:
why 15000 here and not 2k?

Reply:
Ah I was testing it out. I was going for 15k chars as the limit.
On a local run, I can se log lines upto 4k-5k chars. So 15k seems like something that we might possibly log. Raw data should contain upwards of 100k chars, so this might be a reasonable value to detect outliers but not trigger false positives.